xdp/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "agave-xdp"
+description = "Agave XDP implementation"
+version = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = "2021"
+publish = false
+
+[dependencies]
+crossbeam-channel = { workspace = true }
+libc = { workspace = true }
+log = { workspace = true }
+thiserror = { workspace = true }
+
+[target.'cfg(target_os = "linux")'.dependencies]
+aya = { workspace = true }
+caps = { workspace = true }
+
+[dev-dependencies]
+clap = { version = "4.3.0", features = ["derive"] }
```

Comment:
why are we pulling in clap? xdp crate does not have any bin targets.

Reply:
Because I do have a bunch of bins for testing, but I don't plan to commit those. I'll remove clap.

Reply:
removed all dev deps